README.md

```diff
@@ -4,8 +4,44 @@ A simple website badge to show your support for Ukraine.
 
 ### Installation
 ```
-<script src="https://cdn.jsdelivr.net/gh/virae/we-stand-with-ukraine@v1.0.1/badge.js" async></script>
+<script src="https://cdn.jsdelivr.net/gh/austin-dudzik/we-stand-with-ukraine@v1.0.2/badge.js" async></script>
```

Comment:
don't host it on your own repo  leave it as `<script src="https://cdn.jsdelivr.net/gh/virae/we-stand-with-ukraine@v1.0.1/badge.js" async></script>` when he merges the pr it will automatically update it along with the badge so pls change it

Reply:
Good to know. The jsDelivr CDN link has been updated to the original URL.